src/main/java/core/basesyntax/service/impl/FileReaderFromCSVImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.FileReader;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderFromCSVImpl implements FileReader {
+    private final String path;
+
+    public FileReaderFromCSVImpl(String path) {
+        this.path = path;
+    }
+
+    @Override
+    public List<TransactionDto> readFromFile() {
+        List<TransactionDto> list = new ArrayList<>();
```

Comment:
the better variant is, as said Ksenia to me yesterday, that here need to do check to Negative Number, in this method